handler.js

```diff
@@ -237,122 +247,99 @@ module.exports.build_done = (event, context, callback) => {
         target_url: 'https://' + region + '.console.aws.amazon.com/codebuild/home?region=' + region + '#/builds/' + event.build.id + '/view/new',
         context: githubContext,
         description: 'Build ' + buildStatus + '...'
-      }).catch(function(err){
-        console.log(err);
-        context.fail(data);
-      });
+      }).catch(function (err) {
+        console.log(err)
```

Comment:
i think we should update this with the correct call to signal the err here... yeah, realize this wasn't working before anyway :/
```suggestion
        console.log(err)
        callback(err)
```

Reply:
@confusdcodr just this one thing, then good to go

Reply:
Fixed